api-catalog-services/src/test/java/com/ca/mfaas/apicatalog/services/cached/CachedProductFamilyTest.java

```diff
@@ -158,77 +156,31 @@ public void testCallCreationOfContainerThatAlreadyExistsButNothingHasChangedSoNo
     }
 
     @Test
-    public void testCallCreationOfContainerThatAlreadyExistsInstanceInfoHasChangedSoUpdateLastChangeTime() throws InterruptedException {
+    public void testCallCreationOfContainerThatAlreadyExistsInstanceInfoHasChanged() {
         HashMap<String, String> metadata = new HashMap<>();
 
         metadata.put(CATALOG_UI_ID_KEY, "demoapp");
         metadata.put(CATALOG_UI_TITLE_KEY, "Title");
         metadata.put(CATALOG_UI_DESCRIPTION_KEY, "Description");
         metadata.put(CATALOG_UI_VERSION_KEY, "1.0.0");
         InstanceInfo instance = getStandardInstance("service", InstanceInfo.InstanceStatus.UP, metadata);
-        APIContainer originalContainer = service.getContainer("demoapp", instance);
-        Calendar createTimestamp = originalContainer.getLastUpdatedTimestamp();
+        service.getContainer("demoapp", instance);
 
         metadata.put(CATALOG_UI_TITLE_KEY, "Title 2");
         metadata.put(CATALOG_UI_DESCRIPTION_KEY, "Description 2");
         metadata.put(CATALOG_UI_VERSION_KEY, "1.0.1");
         instance = getStandardInstance("service", InstanceInfo.InstanceStatus.UP, metadata);
-        Thread.sleep(100);
 
         APIContainer updatedContainer = service.createContainerFromInstance("demoapp", instance);
-        Calendar updatedTimestamp = updatedContainer.getLastUpdatedTimestamp();
-
-        boolean equals = updatedTimestamp.equals(createTimestamp);
-        assertFalse(equals);
+        checkAPIContainer(updatedContainer, instance);
 
-        metadata.put(CATALOG_UI_TITLE_KEY, "Title 2");
-        metadata.put(CATALOG_UI_DESCRIPTION_KEY, "Description 2");
-        metadata.put(CATALOG_UI_VERSION_KEY, "2.0.0");
+        metadata.put(CATALOG_UI_TITLE_KEY, "Title 3");
+        metadata.put(CATALOG_UI_DESCRIPTION_KEY, "Description 3");
+        metadata.put(CATALOG_UI_VERSION_KEY, "3.0.0");
         instance = getStandardInstance("service", InstanceInfo.InstanceStatus.UP, metadata);
-        Thread.sleep(100);
 
         service.updateContainerFromInstance("demoapp", instance);
-        Calendar retrievedTimestamp = updatedContainer.getLastUpdatedTimestamp();
-
-        equals = updatedTimestamp.equals(retrievedTimestamp);
-        assertFalse(equals);
-    }
-
-    @Test
-    public void testCallCreationOfContainerForNullVersion() throws InterruptedException {
```

Comment:
It was the duplicate of testCallCreationOfContainerThatAlreadyExistsInstanceInfoHasChangedSoUpdateLastChangeTime()